packages/mui-material/src/Select/Select.test.js

```diff
@@ -484,6 +485,61 @@ describe('<Select />', () => {
       expect(getAllByRole('option')[1]).to.have.attribute('aria-selected', 'true');
     });
 
+    describe('when the first child is a ListSubheader', () => {
```

Comment:
Please add a test with two headers at the top

Reply:
You mean this way 
```js
describe('when the first child is not a selectable option', () => {
  describe('when is a ListSubheader', () => {
    it('first selectable option is focused to use the arrow', () => {
    });
  })
  describe('when is a MenuItem disabled', () => {
    it('first selectable option is focused to use the arrow', () => {
    });
  })
});
```

Reply:
I mean to test the following case:

```jsx
<Select defaultValue="" open>
  <ListSubheader>Empty category</ListSubheader>
  <ListSubheader>Category 1</ListSubheader>
  <MenuItem value={1}>Option 1</MenuItem>
  <MenuItem value={2}>Option 2</MenuItem>
  <ListSubheader>Category 2</ListSubheader>
  <MenuItem value={3}>Option 3</MenuItem>
  <MenuItem value={4}>Option 4</MenuItem>
</Select>